portalnet/src/storage.rs

```diff
@@ -754,45 +758,45 @@ impl PortalStorage {
 
 #[derive(Debug)]
 struct StorageMetrics {
-    content_storage_usage_kb: Gauge,
-    total_storage_usage_kb: Gauge,
-    storage_capacity_kb: Gauge,
+    content_storage_usage_mb: Gauge,
+    total_storage_usage_mb: Gauge,
+    storage_capacity_mb: Gauge,
```

Comment:
Sorry, I gave the suggestion to switch the suffixes to `_mb` before I read [best practices](https://prometheus.io/docs/practices/naming/#base-units).

I think the best option would be to switch it to `_bytes` in the metric, and convert from MB for the CLI arg, and to MB for the heartbeat log.

I'm going to go in eventually and follow more of these metrics to best practices (like using labels for the protocol, instead of adding it onto the metric name), so either way it will get done eventually. But if you feel up to making the switch that would be appreciated.

Reply:
went ahead and took care of this, using `bytes` for the individual gauges and then converting to `mb` in the summary